flatcar-release-mirror.sh

```diff
@@ -124,13 +125,19 @@ download_folder() {
         if [[ -z "$above_version" ]] || [[ -z "$is_version" ]] || [ "$version" -gt "$above_version" ]; then
           mkdir -p "$link"
           cd "$link"
-          download_folder "${url}$link" || return 1
+          download_folder "${url}$link" "$arch" || return 1
           cd ..
         else
           echo "Skipping folder version: $link" >> "$logfile"
         fi
       else
-        download_file "${url}$link" "$link" || return 1
+        if [[ ! $url =~ "flatcar-linux.net/"$ ]]; then
```

Comment:
Thanks for the PR, can you explain why exactly this string is matched?

Reply:
I had test again, these line is no need anymore.
Nice catch!